crates/sncast/src/response/structs.rs

```diff
@@ -144,3 +144,17 @@ pub struct TransactionStatusResponse {
 }
 
 impl CommandResponse for TransactionStatusResponse {}
+
+#[derive(Serialize)]
```

Comment:
It's just a `Result`, isn't it?

Reply:
I have removed the `verification_status` field, as you pointed out that it's similar to `Result<>`. Now, `VerifyResponse` allows us to easily implement the `CommandResponse` trait, maintaining consistency with other response structures such as `ScriptInitResponse`.